plugins/UserCountry/lang/en.json

```
@@ -1,51 +1,51 @@
-        "PluginDescription": "Reports the location of your visitors: country, region, city and geographic coordinates (latitude\/longitude).",
+        "NoDataForGeoIPReport1": "No data for this report, as there is either no location data available, or visitor IP addresses cannot be geolocated.",
+        "NoDataForGeoIPReport2": "%1$Change these settings%2$s to get accurate geolocation, and a %3$scity-level database%4$s.",
+        "NoProviders": "No additional geolocation providers available. %1$sDB-IP%2$s is recommended, but requires the GeoIp2 plugin. (You can also install a third-party geolocation plugin from the marketplace.)",
+        "PluginDescription": "Reports the location of your visitors: country, region, city and geographic coordinates (latitude and longitude).",
         "Region": "Region",
         "SubmenuLocations": "Locations",
-        "TestIPLocatorFailed": "Matomo tried checking the location of a known IP address (%1$s), but your server returned no information. This provider may not be configured correctly (the geolocation database may be named incorrectly or located in the wrong directory, for instance).",
-        "ToGeolocateOldVisits": "To get location data for your old visits, use the script described %1$shere%2$s.",
-        "WidgetLocation": "Visitor Location",
-        "GeoIpDbIpAccuracyNote": "Note: the DBIP databases are free and can be downloaded automatically, but geolocation results (specifically city results) are not as accurate as MaxMind's. MaxMind, however, requires that you create an account even for the free database. If you want to use MaxMind's geolocation database, you can start the process %1$shere%2$s",
-        "MaxMindLinkExplanation": "If you are using MaxMind's geolocation databases and you do not already know how to generate your download URL, %1$sclick here to learn how%2$s.",
-        "LocationsSubcategoryHelp": "The Locations section is the best way to find out where people are when they visit your site. It reveals the countries, continents, regions, cities that you visitors come from, in tables and map form. Additionally, you can see what language their browser is set to, helping identify international visitors in alternative locations."
+        "TestIPLocatorFailed": "Couldn't find the location of the IP address \"%1$s\". Maybe this provider is not set up correctly, the geolocation database is incorrectly named, or in the wrong folder?",
+        "ToGeolocateOldVisits": "Use %1$this script%2$s to get location data for your old visits.",
+        "WidgetLocation": "Visitor location",
+        "GeoIpDbIpAccuracyNote": "Note: the DB-IP geolocation databases are gratis and can be auto-downloaded, but not as accurate as MaxMind's for cities. MaxMind requires %1$making an account%2$s.",
```

Isn't the `are` missing? Otherwise it might sound like the download isn't as accurate 🤔 

```suggestion
        "GeoIpDbIpAccuracyNote": "Note: the DB-IP geolocation databases are gratis and can be auto-downloaded, but are not as accurate as MaxMind's for cities. MaxMind requires %1$making an account%2$s.",
```

---

I was able to do without, and got rid of an extra "MaxMind".